src/TransformOperationExecutor.ts

```diff
@@ -96,9 +96,13 @@ export class TransformOperationExecutor {
             }
             if (value === null || value === undefined)
                 return value;
-
             return new Date(value);
 
+        } else if (testForBuffer() && (targetType === Buffer || value instanceof Buffer) && !isMap) {
```

Comment:
I don't know if this is a good idea. 
Buffer is a module from node. This would make the whole library incompatible for execution in the browser.

Reply:
I aggree

Reply:
In #53 we discussed a more generic solution. For now we keep this solution until we have the more generic one.